src/core/meshcop/commissioner.hpp

```diff
@@ -209,6 +209,9 @@ class Commissioner
         kRemoveJoinerDelay    = 20,     ///< Delay to remove successfully joined joiner
     };
 
+    void AddCoapResources();
+    void RemoveCoapResources();
```

Comment:
nit: All void functions or methods shall explicitly declare and specify the void type keyword.